sebtm/inwx-api-bundle/1.0/config/packages/inwx_api.yaml

```diff
@@ -0,0 +1,8 @@
+# Read the documentation: https://github.com/SebTM/inwx-api-bundle
+inwx_api:
+    debug: false
+    environment: development
```

Comment:
You should remove these two and maybe read the default from `%kernel.debug%`, `%kernel.environment%`, right?

Reply:
I understand the first point with removing the not needed defaults, but I think there are cases you want to enable debugging and/or switch to different environment regardless "%kernel.*%" settings. I would suggest to set them to the "%kernel.*" as default but keep them independent changeable.

Would you be fine with this?

Reply:
@Nyholm Ping, would you be fine with this as proposed?

Reply:
Yes